### PR TITLE
WIFI-2604 wlan-ap-factory-reset.sh calls DigiCert

### PR DESCRIPTION
**Pull request by RickSommerville**
Signed-off-by: Rick Sommerville <rick.sommerville@netexperience.com>

### CODE REVIEW

File: feeds/wlan-ap/opensync/src/platform/openwrt/rootfs/common/etc/init.d/opensync [hunk 1]
```diff
@@ -40,17 +40,20 @@ start_service() {
...
-            wlan_ap_redirector.sh ${redirector}
+        [[ -f /usr/opensync/certs/redirector.txt ]] && saved_redirector=$(cat /usr/opensync/certs/redirector.txt | tr -d '\r\n')
+        logger -t opensync "Contacting DigiCert for redirector address"
+        wlan_ap_redirector.sh
+        new_redirector=$(uci get system.tip.redirector)
+        if [ -z "$new_redirector" && -n "$saved_redirector" ]; then
+            logger -t opensync "No response from DigiCert, using saved redirector address ${saved_redirector}"
+            wlan_ap_redirector.sh ${saved_redirector}
+        else 
+            logger -t opensync "DigiCert returned redirector address ${new_redirector}"
```
Comment (chaitanyagodavarthi) [inline feeds/wlan-ap/opensync/src/platform/openwrt/rootfs/common/etc/init.d/opensync:51]: Where do we apply the new_redirector?

**Review by chaitanyagodavarthi — COMMENTED** [review]
.

File: feeds/wlan-ap/opensync/src/platform/openwrt/rootfs/common/etc/init.d/opensync (hunk 1 above)
Comment (RickSommerville) [reply feeds/wlan-ap/opensync/src/platform/openwrt/rootfs/common/etc/init.d/opensync:51]: We don't, it was already discovered and applied by the call to wlan_ap_redirector.sh at line 45.  If it hadn't been discovered and applied we'd use the saved redirector at line 49.

**Review by chaitanyagodavarthi — APPROVED** [review]
Looks good

**Comment by RickSommerville** [comment]
Merged 17918705

**Pull request closed without merging**